src/components/Layout.jsx

```diff
@@ -1,15 +1,25 @@
-import React from 'react'
-
-import {Home} from '../pages'
+import React, { useState } from 'react'
+import {Home, ProductList} from '../pages'
 import {Footer, Navbar} from './index'
 
 const Layout = () => {
+
+  const [pageToRender, setPageToRender] = useState('Home')
```

Comment:
Be careful with string comparisons in JS apps, may be bugs because miss writing, a best practice is to have JS objects as dictionaries, so you can make something like this:
`const LAYOUT_VIEW = {
HOME: 'Home',
PRODUCTS: 'Products'
}`
And then you can make use of it as: `useState(LAYOUT_VIEW.HOME)`